provider/pkg/schema/docs_schema.go

```diff
@@ -0,0 +1,49 @@
+package schema
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type Docs struct {
+	Types map[string]DocsTypes `json:"Types"`
+}
+
+type DocsTypes struct {
+	Properties  DocsProperties    `json:"properties"`
+	Attributes  map[string]string `json:"attributes"`
+	Description string            `json:"description"`
+}
+
+type DocsProperties map[string]string
+
+func ReadCloudFormationDocsFile(filePath string) (*Docs, error) {
+	docs := Docs{}
+	in, err := os.Open(filePath)
+	if err != nil {
+		return nil, fmt.Errorf("error opening file: %v", err)
+	}
+	defer in.Close()
+
+	dec := json.NewDecoder(in)
+	if err := dec.Decode(&docs); err != nil {
+		return nil, fmt.Errorf("error decoding: %v", err)
+	}
+	return &docs, nil
+}
+
+func (d *Docs) GetPropertyDesc(typeName, propName string) (string, bool) {
+	if typ, exists := d.Types[typeName]; exists {
+		if propName == "" {
+			return typ.Description, true
+		}
+		if prop, ok := typ.Properties[propName]; ok {
```

Comment:
Curious, no difference between Properties and Attributes for the purposes here? It always prefers Properties?

Reply:
I'll create a separate issue to track this. There was some additional complexity for figuring out which were attributes and which were properties, and the attributes in the CFN docs were structured differently.

There is a chance that some properties and attribute overlap at the same path in the schema and then we may have the wrong description for the time being.